lib/chef/provider/aws_rds_subnet_group.rb

```diff
@@ -0,0 +1,84 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsSubnetGroup < Chef::Provisioning::AWSDriver::AWSProvider
+
+  def create_aws_object
+    converge_by "create new RDS Subnet Group #{new_resource.name} in #{region}"do
+      driver.create_db_subnet_group(desired_options)
+    end
+  end
+
+  def destroy_aws_object(object)
+    converge_by "delete RDS Subnet Group #{new_resource.name} in #{region}" do
+      driver.delete_db_subnet_group(db_subnet_group_name: new_resource.db_subnet_group_name)
+    end
+  end
+
+  def update_aws_object(object)
+    updates = required_updates(object)
+    if ! updates.empty?
+      converge_by updates do
```

Comment:
I like making an update string so we get proper output, but don't we need a `updates.join("\n")` so the output appears on a new line?  Or does the `converge_by` handle this already?  What does the output look by when multiple attributes are updated?